From 80a5d4cc2379eb1beb200c4437affa4d31d65c7b Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Mon, 24 May 2021 11:36:41 +0300
Subject: [PATCH 04/10] dts: s32g274ardb2: Disable I2C1

commit 6c1ff4ebe413f5be2e6b07c22148d75612ef9a54 from
https://source.codeaurora.org/external/autobsps32/linux

I2C1 shouldn't be enabled as its pins are not exposed on RDB2 board.

Upstream-Status: pending

Issue: ALB-7206
Signed-off-by: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb2.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb2.dts b/arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb2.dts
index a1cb46f94552..60cac789b7be 100644
--- a/arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-s32g274a-rdb2.dts
@@ -99,6 +99,10 @@ rtc@51 {
 	};
 };
 
+&i2c1 {
+	status = "disabled";
+};
+
 &spi5 {
 	/* 1 hardware cs and 1 gpio cs*/
 	spi-num-chipselects = <1>;
-- 
2.25.1

